dlc/Cargo.toml

```diff
@@ -9,10 +9,11 @@ version = "0.4.0"
 
 [dependencies]
 bitcoin = {version = "0.29.2"}
+lightning = {version = "0.0.117"}
```

Comment:
I would rather this crate to stay very simple and not import `lightning` unless necessary. Actually is there a reason that you put `ids.rs` here and not in `dlc-manager`? It doesn't seem to be used within this crate.

Reply:
> Actually is there a reason that you put ids.rs here and not in dlc-manager? It doesn't seem to be used within this crate.

I was also hesitant about this. But it seemed to me like IDs belong in a core crate.

The reason why I didn't want to put them in `dlc-manager` was because IDs are used in `dlc-messages`, meaning that `dlc-messages` would have to depend on `dlc-manager` which seems weird. And, more importantly, that would create a circular dependency.

> I would rather this crate to stay very simple and not import lightning unless necessary.

I considered adding a feature flag to the `dlc` crate, but it was getting a bit tedious so I dropped it. Should I try that again?

Reply:
Yeah a circular dependency wouldn't be good. What do you think about just not using these types for the `dlc-messages` crate? I guess it's not that a big deal to have it in `dlc` though, so I'll leave it up to you.